www/services/HistoricFilterService.class.php

```diff
@@ -73,22 +73,29 @@ function getAllEntriesAsRows() {
         $html = "";
         //TODO may need to make a function that makes a JS array to hold the info
         foreach ($allModels as $model) {
-            $objectRowID = "14" . strval($model -> getIdHistoricFilter());
+            $idHistoricFilter = strval($model -> getIdHistoricFilter());
+            $objectRowID = "14" . $idHistoricFilter;
             $editAndDelete = "</td><td><button onclick='updateHistoricFilter("
                 . $objectRowID . ","
                 . $model -> getIdHistoricFilter()
                 . ")'>Update</button>"
-                . "</td><td><button onclick=" . '"deleteHistoricFilter('
-                . $model -> getIdHistoricFilter()
-                . ')"> Delete</button>';
+                . "</td>";
+
+            if($idHistoricFilter != "0") {
+                $editAndDelete = $editAndDelete . "<td><button onclick="
```

Comment:
@bfj5889 i do not start a new table row anywhere. i just do not add a new table cell, `<td></td>` with the delete button if it is a no historic filter

Reply:
should i add the cell without the button or just do not add the cell altogether?